compiler/src/iree/compiler/InputConversion/StableHLO/Rewriters.h

```diff
@@ -11,6 +11,10 @@
 
 namespace mlir::iree_compiler::stablehlo {
 
+//===----------------------------------------------------------------------===//
+// General StableHLO/CHLO lowering patterns.
+//===----------------------------------------------------------------------===//
```

Comment:
Just StableHLO, right? Not also CHLO?

Reply:
We haven't fleshed out chlo support yet -- I expect that once we add support to more chlo, we will either decide to keep them together or to separate stablehlo from chlo lowering. Because of the overlap in ops, the former seems more likely at the moment.

Reply:
I'm still a bit confused by the presence of CHLO here. The appeal of StableHLO is that it is stable, and presumably something we can depend on in isolation. I'd expect a program containing _just_ StableHLO to be forwards compatible ("stable"), while a program containing StableHLO and a mix of other dialects (arith, TOSA, ml_program, CHLO, etc.) to not _necessarily_ be forwards compatible. Including other dialects in this input conversion dilutes that property.

Reply:
Ah... a little concerning, but okay...

I'm wondering if we'd want to isolate those into a different directory / set of passes to make them easier to prune later.